demisto_sdk/commands/test_content/test_modeling_rule/test_modeling_rule.py

```diff
@@ -1483,6 +1796,69 @@ def test_modeling_rule(
         "-dd",
         help="Deletion of the existing dataset from the tenant. Default: False.",
     ),
+    service_account: Optional[str] = typer.Option(
+        None,
+        "-sa",
+        "--service_account",
+        envvar="GCP_SERVICE_ACCOUNT",
+        help="GCP service account.",
+        show_default=False,
+    ),
+    cloud_machine_ids: str = typer.Option(
```

Comment:
do we need this?

Reply:
yes, the design is both to get the list of machines and the general config file containing all the machines.

Reply:
yes, the design is both to get the list of machines and the general config file containing all the machines.

Reply:
OK, but I don't see where in the code you use this, since we get the relevant machine IDs as part of the machine_assignment file.

Reply:
you're correct, removing it